contrib/prometheus/prometheus-all.json

```diff
@@ -72,11 +86,25 @@
                         ]
                     },
                     {
-                        "name": "kubectl",
-                        "image": "gcr.io/google_containers/kubectl:v0.18.0-350-gfb3305edcf6c1a",
-                        "args": [
-                            "proxy", "-p", "8001", "--api-prefix=/"
+                        "image": "jayunit100/prompush:0.2.0",
+                        "name": "pushgateway",
+                        "ports": [
+                            {
+                                "containerPort": 9091,
+                                "hostPort": 9091,
+                                "protocol": "TCP"
+                            }
                         ]
+                    },
+                    {
+                        "args": [
+                            "proxy",
+                            "-p",
+                            "8001",
+                            "--api-prefix=/"
+                        ],
+                        "image": "gcr.io/google_containers/kubectl:v0.18.0-350-gfb3305edcf6c1a",
+                        "name": "kubectl"
```

Comment:
maybe we can add this to verify-fmt ? its a good convention that your suggestion.  ill fix it now